Core/clim-core/clim-core.asd

```diff
@@ -1,6 +1,6 @@
 
 (defsystem #:clim-core
-  :depends-on (#:clim-basic #:clim-postscript)
+  :depends-on (#:clim-basic #:clim-postscript #:clim-pdf)
   :components
```

Comment:
is it part of the core though? I'd put it as a dependency of mcclim-extensions, since it's not defined in clim itself